static/scripts/homework/edit.js

```diff
@@ -80,4 +71,16 @@ window.addEventListener('DOMContentLoaded', () => {
 			$('#teamsize').addClass('hidden-xl-down');
 		}
 	});
+
+	$('#homework-submit-btn').on('click', (event) => {
+		event.preventDefault();
+		const availableDate = $('#availableDate').val().split(' ');
+		const dueDate = $('#dueDate').val().split(' ');
+		if (dueDate[0] === '' || dueDate > availableDate) {
```

Comment:
should check for length first? Also, the dueDate and availableDate are now arrays, not date. What do we compare?

Reply:
not sure about the length and if that is needed. but I saw that method to compare the different entries of an array somewhere else and I found that suitable for our case of the date. Not really sure how to do it in a different way